pkg/app/server.go

```diff
@@ -85,11 +87,11 @@ func RunKubeStateMetricsWrapper(ctx context.Context, opts *options.Options, fact
 // RunKubeStateMetrics will build and run the kube-state-metrics.
 // Any out-of-tree custom resource metrics could be registered by newing a registry factory
 // which implements customresource.RegistryFactory and pass all factories into this function.
-func RunKubeStateMetrics(ctx context.Context, opts *options.Options, factories ...customresource.RegistryFactory) error {
+func RunKubeStateMetrics(ctx context.Context, opts *options.Options) error {
+	var factories []customresource.RegistryFactory
```

Comment:
could we move that closer to where it is used in the function?

Reply:
Done!